config/v1/types_infrastructure.go

```diff
@@ -231,6 +234,10 @@ type PlatformSpec struct {
 	// EquinixMetal contains settings specific to the Equinix Metal infrastructure provider.
 	// +optional
 	EquinixMetal *EquinixMetalPlatformSpec `json:"equinixMetal,omitempty"`
+
+	// PowerVS contains settings specific to the IBM Power Systems Virtual Servers infrastructure provider.
+	// +optional
+	PowerVS *PowerVSPlatformSpec `json:"powervs,omitempty"`
```

Comment:
not `powerVS`?

Reply:
we are just following the preexisting providers what they are following except `equinixMetal`, no hard rule as such.

Reply:
if it's okay, i'd like to leave as all lowercase. i personally thought it was odd that equixMetal was the only provider that had any caps. it seems everything else is all lowercase. using oVirt as an example specifically since it's the only one that would make a lot of sense to type as `oVirt`. also to be consistent with what we're doing in other repos.

Reply:
alll the others are wrong :) APIs use camlCase.